src/writing-policies/spec/02-settings.md

```diff
@@ -0,0 +1,37 @@
+# Policy settings
+
+Policy behaviour is not set in stone, it can be used by providing configuration
+details to the policy at runtime. The policy author has full freedom to define
+the structure of his policy settings.
+
+Kubewarden takes care of serializing the policy Settings into JSON and provide
+them to the policy every time it is invoked.
+
+## Settings validation
+
+Some policies might want to validate the settings a user provides to ensure
+they are correct.
+
+Each policy must register a waPC function called `validate_settings` that
+takes care of validating the policy settings.
+
+The `validate_settings` function receives as input a JSON representation of
+the settings provided by the user. The function validates them and returns
+as a response a `SettingsValidationResponse` object.
```

Comment:
Should we name the object here? When using a specific SDK naming it sounds fine, but talking about raw JSON, seems strange to name a unnamed raw JSON struct.